From ed7577cb855337d9a93e66fa0eb73eb604fe8f04 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@joyent.com>
Date: Mon, 14 Aug 2017 21:35:31 +0000
Subject: [PATCH] OS-6295 t4nex driver missing pciex1425,6405 Reviewed by:
 Patrick Mooney <patrick.mooney@joyent.com> Reviewed by: Joshua M. Clulow
 <jmc@joyent.com> Approved by: Patrick Mooney <patrick.mooney@joyent.com>

---
 overlay/generic/etc/driver_aliases | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/overlay/generic/etc/driver_aliases b/overlay/generic/etc/driver_aliases
index f9d9080e..eda1c7b9 100644
--- a/overlay/generic/etc/driver_aliases
+++ b/overlay/generic/etc/driver_aliases
@@ -1162,7 +1162,7 @@ t4nex "pciex1425,6401"
 t4nex "pciex1425,6402"
 t4nex "pciex1425,6403"
 t4nex "pciex1425,6404"
-t4nex "pciex1425,6404"
+t4nex "pciex1425,6405"
 t4nex "pciex1425,6406"
 t4nex "pciex1425,6407"
 t4nex "pciex1425,6408"
-- 
2.21.0

